### ISSUE TITLE
Improper use of aria-labelledby

### ISSUE THREAD

**Issue by hectorcorrea** [projectblacklight/blacklight#1776]
HTML attribute "aria-labelledby" in https://github.com/projectblacklight/blacklight/blob/master/app/views/shared/_modal.html.erb#L1 (_ajax_modal.html.erb in previous versions) points to an undefined ID and tools like [WAVE](http://wave.webaim.org/) flag it as an accessibility error (see attached screenshot)

![screen shot 2017-09-05 at 12 14 04 pm](https://user-images.githubusercontent.com/568286/30071170-cdeb1cee-9233-11e7-97cb-756ca52e91e5.png)

I suggest to either remove the "aria-labelledby" from this element since we are not using it properly or add the proper elements inside the HTML e.g. 

```
<div id="ajax-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="modal menu" aria-hidden="true">
  <div id="modal">
    <div id="menu"></div>
  </div>
    <div class="modal-dialog">
    <div class="modal-content">
    </div>
  </div>
</div>
```

**Comment by cbeer** [comment]
I suspect we copied this example directly from bootstrap. It seems like they've fixed it upstream, and we should just copy their recommendation.

**Comment by sandbergja** [comment]
@cbeer: could you point me toward the recommendation you're referencing?  I'd be happy to create a pull request that makes this partial more in line with the recommendation (or, alternatively, just removes the aria-labelledby per @hectorcorrea's suggestion).

**Comment by barmintor** [comment]
In the Bootstrap 4 docs: https://getbootstrap.com/docs/4.1/components/modal/
In the Bootstrap 3 docs: https://getbootstrap.com/docs/3.3/javascript/#modals

I don't see `calss="menu"` anywhere, so removal seems like best path here?
